### PR TITLE
css-selector is an optional dependency of the browser-kit

### PR DESCRIPTION
**Pull request by dbu**
restructured the testing introduction to not end up with an error in the first functional test example

### CODE REVIEW

File: testing.rst [hunk 1]
```diff
@@ -197,8 +196,25 @@ be used to select elements in the response, click on links and submit forms.
     The ``Crawler`` only works when the response is an XML or an HTML document.
     To get the raw content response, call ``$client->getResponse()->getContent()``.
 
-Click on a link by first selecting it with the crawler using either an XPath
-expression or a CSS selector, then use the client to click on it. For example::
+The crawler integrates with the ``symfony/css-selector`` component to give you the
+power of CSS selectors to find content in a page. To install the css selector
```
Comment (javiereguiluz) [inline testing.rst]: Minor thing: `css selector` -> `CSS selector`

**Review by javiereguiluz — COMMENTED** [review]
I like this a lot. I left some minor comments. Thanks.

File: testing.rst (hunk 1 above)
Comment (dbu) [reply testing.rst]: fixed

**Review by javiereguiluz — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks David.

**Pull request merged**